Tests/AWSLambdaRuntimeCoreTests/LambdaLifecycleTest.swift

```diff
@@ -68,6 +74,12 @@ class LambdaLifecycleTest: XCTestCase {
         defer { XCTAssertNoThrow(try server.stop().wait()) }
         let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
         defer { XCTAssertNoThrow(try eventLoopGroup.syncShutdownGracefully()) }
+        
+        let serviceLifecycle = ServiceLifecycle(configuration: .init(shutdownSignal: [], installBacktrace: false))
+        defer {
+            serviceLifecycle.shutdown()
+            serviceLifecycle.wait()
+        }
```

Comment:
I wonder if we can now simplify the shutdown logic a bit by relying more on the users using lifecycle